food-truck-frontend/.env

```diff
@@ -1,2 +1,2 @@
 FOOD_TRUCK_API_URL="https://spring-kings.herokuapp.com"
-GOOGLE_MAPS_API_KEY=AIzaSyAntHzd2366GW2dvpu8BUCT4emcUN9KGsY
```

Comment:
Was this the production key? I must have uncommitted it by mistake during merge conflicts.

Reply:
Yeah, that's the production key. We'll need to put it back.